package.json

```diff
@@ -21,7 +21,7 @@
     "error-polyfill": "^0.1.2",
     "local-storage": "^2.0.0",
     "near-api-js": "^0.45.1",
-    "near-social-vm": "NearSocial/VM#1.2.0",
+    "near-social-vm": "near/nearsocial-vm#analytics",
```

Comment:
@charleslavon We can't switch over to this version of the VM - it doesn't have the iframe resizer changes. Are we permanently diverging from https://github.com/NearSocial/VM? CC @mpeterdev

Reply:
Good catch. I didn't intend to include this change. I think the only reason that might warrant switching to our fork is whether or not we get the Ether.js changes merged back into nearsocial/vm

Reply:
This PR needs to be approved/merged first https://github.com/near/nearsocial-vm/pull/2

Reply:
Sounds good. Looks like if we need to switch to `near/nearsocial-vm` we just need someone to sync our fork with the upstream changes (so we can maintain changes like the iframe resizer). From what I can tell, that PR you linked doesn't have the latest.